app/jobs/govbox/check_messages_mapping_job.rb

```diff
@@ -3,17 +3,17 @@ class CheckMessagesMappingJob < ApplicationJob
     queue_as :default
 
     def perform
-      unmapped_govbox_messages_sql =
-      "SELECT gm.id FROM govbox_messages gm JOIN govbox_folders gf ON gm.folder_id = gf.id
-        WHERE gm.delivered_at > '#{1.months.ago}'
-        AND NOT EXISTS (
-        SELECT 1 FROM messages m
-        JOIN message_threads mt ON m.message_thread_id = mt.id
-        WHERE m.uuid = gm.message_id
-        AND mt.box_id = gf.box_id
-        LIMIT 1
-       )"
-      unmapped_govbox_messages = ActiveRecord::Base.connection.execute(unmapped_govbox_messages_sql)
+      unmapped_govbox_messages = Govbox::Message.find_by_sql(
```

Comment:
Toto je funkcne ok, ak by si sa chcela inspirovat ako to spravit cez AR/arel tak kuk sem. https://github.com/slovensko-digital/govbox-pro/blob/bb142029a790b6964efc940bec09c9c1f8189a07/app/models/user.rb#L49-L55

Reply:
Upravila som